src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -634,7 +637,13 @@ private async Task<List<LaunchCommand>> GetInitializeCommands()
 
             // On Windows ';' appears to correctly works as a path seperator and from the documentation, it is ':' on unix
             string pathEntrySeperator = _launchOptions.UseUnixSymbolPaths ? ":" : ";";
```

Comment:
>";"; [](http://example.com/codeflow?start=82&length=4)

Do you know if this is correct for cygwin?

Reply:
Cygwin is assuming everything is based in linux, so it checks for `:`.